cypress/e2e/ResumeParserAPI/pricing.cy.js

```diff
@@ -0,0 +1,123 @@
+import { RESUME_PARSER_API } from '../../resources/data';
+
+describe('Resume Parser API Pricing page', () => {
+    before(() => {
+        cy.visit(RESUME_PARSER_API);
+    });
+
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        // Check if the "Accept Cookies" button is visible or exists
+        cy.AcceptCookies();
+    });
+
+    it('2. Test to check "Resume Parser API" title is present', () => {
+        cy.componentVisiblityCheck('h1, h2, h3, h4', 'Resume Parser API');
+    });
+
+    // Test to check visibility of the "Free Plan" title and its price
+    it('3. Test to check "Free Plan" title and price are present', () => {
+        // Verify "Free Plan" title visibility and content
+        cy.get('.card-header .d-block.h3')
+            .should('be.visible')
+            .and('contain.text', 'Free Plan');
+
+        // Verify "Free Plan" price visibility and content
+        cy.get('.plan-price.font-size-4.font-weight-bold.text-primary')
+            .should('be.visible')
+            .and('contain.text', '$0.00');
+    });
+
+    // Test to verify the "Subscribe" button visibility and click functionality
+    it('4. Test to click on "Subscribe" button for Free Plan', () => {
+        // Verify that the button is visible and contains the text "Subscribe"
+        cy.get('a.btn.btn-success.btn-block.transition-3d-hover.js-hs-unfold-invoker')
+            .should('be.visible')
+            .and('contain.text', 'Subscribe')
+            .click(); // Perform the click action
+
+        // Check if the sidebar content becomes visible after the click
+        cy.get('#sidebarContent')
+            .should('be.visible');
+
+        cy.get('body').click(0, 0); // Click at the top-left corner of the body to close any opened dropdowns
+    });
+
+    // Test to check the visibility of the "Starter Plan" title and price
+    it('5. Test to check "Starter Plan" title and price are present', () => {
+        // Verify the "Starter Plan" title is visible and contains the correct text
+        cy.get('span.d-block.h3')
+            .should('be.visible')
+            .and('contain.text', 'Starter Plan');
+
+        // Verify the price of the "Starter Plan" is visible and contains the correct amount
+        cy.get('.plan-price.font-size-4.font-weight-bold.text-primary')
+            .should('be.visible')  // Ensure the price element is visible
+            .and('contain.text', '$29.99'); // Verify the price text is correct
+    });
```

Comment:
Test case is missing to check in the starter plan
![image](https://github.com/user-attachments/assets/5550a672-f364-4840-9f76-8530c90ffafb)

Reply:
done